components/coding/studentPortal/AssignmentComponent.tsx

```diff
@@ -8,9 +8,13 @@ export type Hint = {
   icon?: string;
   link?: string;
 };
+
 export enum Stage {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
i think you can actually just store this in the template and you won't get this error because it's being used there

eslint is here to make us write better code so let's not disable unless we have to

Reply:
I tried that and it still didn't read it lol. said the variable was not being used, but I will move it to template regardless